src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -3046,6 +3046,7 @@ void MessageHelper::SendOnStatusUpdate(const std::string& status,
 }
 
 void MessageHelper::SendGetSystemInfoRequest(ApplicationManager& app_mngr) {
+  SDL_LOG_AUTO_TRACE();
```

Comment:
@ychernysheva I think this is not related to fix and can be removed

Reply:
Debug is removed in 781d890e25f60d9a87b1f4bc8a8d1cecd8f857b0